### PR TITLE
add temp_folder:create task

### LINKED ISSUE #7: Task for creating temp directory
The gem is littered with code that creates the local temp directory (`FileUtils.mkdir_p 'temp'`)

This directory creation should be outsourced in a specific rake task that is invoked from other tasks.

### PR DESCRIPTION
**Pull request by klyonrad**
This adds a temp folder tasks and invokes it from other tasks

Closes #7

### CODE REVIEW

File: lib/dkdeploy/tasks/temp_folder.rake [hunk 1]
```diff
@@ -0,0 +1,10 @@
+require 'dkdeploy/constants'
+
+include Capistrano::DSL
+
+namespace :temp_folder do
+  desc 'Create local temporary directory'
+  task :create do
+    FileUtils.mkdir_p local_dump_path
```
Comment (websi) [inline lib/dkdeploy/tasks/temp_folder.rake]: Please use sshkit api. http://www.rubydoc.info/github/capistrano/sshkit/SSHKit%2FDSL%3Arun_locally

Comment (klyonrad) [reply lib/dkdeploy/tasks/temp_folder.rake]: Done.

**Comment by ncreuschling** [comment]
Please rename/move this task into `utils:create_temp_folder`.

**Comment by ncreuschling** [comment]
This needs to be reopened because of broken code. Please fix and resubmit.

**Pull request merged**